crates/viewer/re_viewer_context/src/lib.rs

```diff
@@ -35,58 +35,59 @@ mod viewer_context;
 // TODO(andreas): Move to its own crate?
 pub mod gpu_bridge;
 
-pub use annotations::{
-    AnnotationMap, Annotations, ResolvedAnnotationInfo, ResolvedAnnotationInfos,
+pub use self::{
```

Comment:
your ide went off the rails here? Doesn't make sense to group things under `pub use self::`.